ts/features/bonus/cgn/screens/onboarding/CgnInformationScreen.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import I18n from "../../../../../i18n";
+import { Dispatch } from "../../../../../store/actions/types";
+import { GlobalState } from "../../../../../store/reducers/types";
+import { emptyContextualHelp } from "../../../../../utils/emptyContextualHelp";
+import { availableBonusTypesSelectorFromId } from "../../../bonusVacanze/store/reducers/availableBonusesTypes";
+import { ID_CGN_TYPE } from "../../../bonusVacanze/utils/bonus";
+import BonusInformationComponent from "../../../common/components/BonusInformationComponent";
+import { navigateBack } from "../../../../../store/actions/navigation";
+
+export type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+/**
+ * This Screen shows all the information about the bpd program, with the rules and t&c.
+ */
+const CgnInformationScreen: React.FunctionComponent<Props> = (props: Props) => {
+  const onConfirm = () => props.userActivateCgn();
+
+  return (
+    <>
+      {props.bonus ? (
+        <BonusInformationComponent
+          primaryCtaText={I18n.t("bonus.bpd.cta.activeBonus")}
```

Comment:
```suggestion
          primaryCtaText={I18n.t("bonus.cgn.cta.activeBonus")}
```

Reply:
I have just added this component as a draft (test were failing cause the CGN navigator had no routes inside) I will create relative ctas/texts and minor fixes on it's specific story https://www.pivotaltracker.com/story/show/176443163

Reply:
@Undermaken I've yet included this fix in this PR with bc0e2f2

If you think everything's ok you can merge this PR